<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>measurement_task</class>
 <widget class="QWidget" name="measurement_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,9,0">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="path_lineedit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>文件夹</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>配置：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>过滤器</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>测量任务：</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget_2"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,0">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>测量任务</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="file_pashbutton">
            <property name="text">
             <string>文件夹</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="file_lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
